tests/setupTest.ts

```diff
@@ -79,3 +90,114 @@ vi.mock("$app/stores", (): typeof stores => {
         updated,
     };
 });
+
+// This type takes the SnowballRClient interface and returns a new type where each function returns a MockUnaryCall i.e. a mocked backend API
```

Comment:
```suggestion
// This type takes the SnowballRClient interface and returns a new "interface" type where each api function returns a MockUnaryCall, i.e. a mocked API call, instead of a normal UnaryCall.
```

I needed some time to understand this comment (although I knew, what this type is used for. What do you think of my suggestion? Or do you have other ideas?

Reply:
Sounds better, thank you!